tests/unit/modules/EmailMan/EmailManTest.php

```diff
@@ -189,7 +189,7 @@ public function testis_primary_email_address()
 
         //test with a dummy email set
         $bean->email1 = 'test@test.com';
-        $this->assertEquals(false, $emailMan->is_primary_email_address($bean));
+        $this->assertEquals(true, $emailMan->is_primary_email_address($bean));
```

Comment:
We need to ensure that this is also in the new test framework

Reply:
checked +1